src/pages/lessons/fundamentals/eth-intro-part-2.mdx

```diff
@@ -0,0 +1,279 @@
+</Callout>
+
+We'll save a deep dive into message signing for a future lesson, but here's some pseudocode to give you an idea of the workflow:
+
+```python
+# 1. write a message
+msg = "amanaplanacanalpanama"
+
+# 2. sign it with an account's private key
+pk = b"..."
+signed_message = sign_message(message=msg, private_key=pk)
+
+# 3. send `signed_message` over the wire
+
+# 4. message receiver decodes sender's public address
+sender = decode_message_sender(msg, signed_message.signature)
+print(sender)
+# '0x5ce9454...b9aB12E'
+```
+
+## Web3 account implications
+
+So, we can create Ethereum accounts with surprising ease: offline and separate from any app. Those accounts can be used to sign messages or send various flavors of transactions. What does this mean for app developers?
+
+### Permanent passwords
+
+A harsh reality of this world is that there are no password recovery services for basic account types. If you lose your private key and recovery phrase, you can kiss that account goodbye. Such is the double-edged sword of true ownership. App developers have an ethical obligation to onboard and educate Ethereum newcomers on this reality. (Note: social recovery wallets may improve this user experience and "account abstraction" is one related research effort – topics for another lesson.)
+
+### Onboarding challenges
+
+Introducing new users to Ethereum is complicated. As you've been learning, there are a number of paradigm shifts that aren't immediately obvious. You may have to guide visitors with no Ethereum account yet or users with no ether to pay for transaction fees. The amount of education material required will depend on your audience, but the whole ecosystem will benefit if you're able to onboard new users gracefully.
+
+### Fewer account management features
+
+Given that users create accounts outside of your app, you may discover that your use case requires few or no account management features at all.
+
+### New business models
+
+Data mining isn't going away, but this new account ownership model enables a healthy alternative to the Web 2.0 model in which a company owns every bit of a user's data and sells it to the highest bidder. Ethereum's smart contract platform offers a world of new incentive structures.
+
+### New software architectures
+
+An interesting wrinkle in the definition of your business model will be what to handle onchain vs. offchain. As we've discussed, message signing requires no onchain interaction. There's also nothing stopping you from using a private database for some of your data and the Ethereum blockchain for other bits of data or functionality. There are plenty of tradeoffs to consider: usability, cost, transparency, decentralization, privacy, and so on.
+
+## And breathe
+
+Did all that sink in? Test yourself:
+
+<br />
+<QuizStatusChecker quiz="quiz-eth-intro-2" />
+
```

Comment:
Can you check question 6 of the final quiz, and the fourth answer option, which reads: 'A password for your account' - it reads a bit funny, given it doesn't start with a verb like the other answers' options?

Reply:
I think it's a really nice quiz.

I was a bit worried when I hit question 9 - 'A transaction to deploy a new contract omits which value?'- I read 'emits' as opposed to 'omits', and I thought that it was a very 'specific information' question, and therefore panicked trying to think of the answer :D.

Reply:
The reason I initially got confused is because of 'emit'ting events:
https://ethereum.org/en/developers/tutorials/logging-events-smart-contracts/
I am a believer in tripping up learners, but actually crafting those situations intentionally and thoughtfully. But don't think we should be leaving anyone confused when we are aware of that confusion. Initial confusion is still confusion. 
A synonym for 'omit' could have been found. e.g. 'leave out', 'exclude', etc.